Dockerfile-interactive

```diff
@@ -18,13 +18,16 @@ RUN poetry install
 
 # Update env variables
 ENV PATH /root/.local/bin:$PATH
-ENV PYTHONPATH "${PYTHONPATH}:/interactive"
-ENV PYTHONPATH "${PYTHONPATH}:/metrics"
-ENV PYTHONPATH "${PYTHONPATH}:/common"
-ENV PYTHONPATH "${PYTHONPATH}:/scripts"
+ENV PYTHONPATH "${PYTHONPATH}:/"
+#ENV PYTHONPATH "${PYTHONPATH}:/.streamlit"
+#ENV PYTHONPATH "${PYTHONPATH}:/interactive"
+#ENV PYTHONPATH "${PYTHONPATH}:/metrics"
+#ENV PYTHONPATH "${PYTHONPATH}:/common"
+#ENV PYTHONPATH "${PYTHONPATH}:/scripts"
 
 # Load files from repository
 COPY /api /$APP_NAME/api
+COPY /.streamlit /$APP_NAME/.streamlit
```

Comment:
Maybe we want to copy whole file tree?

Reply:
We want to leave out notebooks and all locally trained models, but I don't plan for more folders to be created.